package.json

```diff
@@ -56,8 +56,11 @@
     "typescript": "^3.5.1"
   },
   "dependencies": {
+    "@types/mailgun-js": "^0.22.3",
     "google-protobuf": "^3.8.0",
     "grpc": "^1.21.1",
+    "https": "^1.0.0",
+    "mailgun-js": "^0.22.0",
```

Comment:
- [ ] Looks like a unused dependencies here.
- [ ] Don't forget to `npm uninstall needle @types/needle`

Also take care to install any `@types/*` dependency as a `devDependency`.

Reply:
Removed all unused!